### ISSUE TITLE
Getting error in Reaction Spam

### ISSUE THREAD

**Issue by Priyanshu766101** [V4NSH4J/discord-mass-DM-GO#456]
Hey, am not able to use the #4 command which is reaction spam. i want to add reaction in existing message. how can i do that please help me.

**Comment by V4NSH4J** [comment]
Can you describe your error more?

**Comment by Priyanshu766101** [comment]
leave that problem. For now am getting error in option #2 - Mass-dm 
![image](https://user-images.githubusercontent.com/99196600/163779752-9fdb1de6-72c1-4590-a603-db86e91312e7.png)

**Comment by Priyanshu766101** [comment]
am getting this error all the time i use the option- #2

**Comment by Priyanshu766101** [comment]
![image](https://user-images.githubusercontent.com/99196600/163780239-db9915eb-177f-49d4-933b-5f377c212f59.png)
see this image

**Comment by Priyanshu766101** [comment]
error code - 0, status - 400 Bad request , message [5163]

**Comment by cateee69** [comment]
you are probably getting captcha on dm

**Comment by cateee69** [comment]
make sure you use latest version

**Comment by Priyanshu766101** [comment]
> 

nup

**Comment by Priyanshu766101** [comment]
> make sure you use latest version

yeah man you are right. i used the latest version & it worked but now am getting one small issue

**Comment by Priyanshu766101** [comment]
> you are probably getting captcha on dm

yeah am getting this issue now...how i can solve that?
